src/systems/userinput/devices/windows-mixed-reality-controller.js

```diff
@@ -0,0 +1,62 @@
+import { paths } from "../paths";
+import { Pose } from "../pose";
+
+export class WindowsMixedRealityControllerDevice {
```

Comment:
This looks fine and is how most of our controllers are done. I rewrote Oculus Go in a more straightforward way which you might consider doing here as well. Haven't 100% decided if its worth changing all of them.